packages/chakra-ui/src/CheckboxWidget/CheckboxWidget.tsx

```diff
@@ -0,0 +1,24 @@
+import { Checkbox } from '@chakra-ui/react'
+import { WidgetProps } from '@rjsf/core'
+import React from 'react'
+
+const CheckboxWidget = (props: WidgetProps) => {
+  const { id, value, disabled, readonly, onChange, onBlur, onFocus } = props
+
+  const _onChange = ({ target: { checked } }: React.ChangeEvent<HTMLInputElement>) => onChange(checked)
+  const _onBlur = ({ target: { value } }: React.FocusEvent<HTMLInputElement | any>) => onBlur(id, value)
+  const _onFocus = ({ target: { value } }: React.FocusEvent<HTMLInputElement | any>) => onFocus(id, value)
+
```

Comment:
The title here is not rendering properly for checkboxes (it should say "Done?"):

![image](https://user-images.githubusercontent.com/1689183/111926894-0be87800-8a85-11eb-8cd7-f2166e191bae.png)

Here's how it (correctly) renders for the default theme:

![image](https://user-images.githubusercontent.com/1689183/111926905-1a369400-8a85-11eb-98bc-5f5f3009af73.png)